src/test/groovy/com/palantir/gradle/versions/ConsistentVersionsPluginIntegrationSpec.groovy

```diff
@@ -115,34 +115,6 @@ class ConsistentVersionsPluginIntegrationSpec extends IntegrationSpec {
         gradleVersionNumber << GRADLE_VERSIONS
     }
 
-    // TODO(dsanduleac): should remove this since this functionality doesn't fully work anyway, and we are
```

Comment:
This test was redundant since it stopped testing `compile` configuration as a result of automated migration.